### ISSUE TITLE
Incorrect next_bit assignment?

### ISSUE THREAD

**Issue by randyrossi** [bggardner/simplisafe-rf#9]
https://github.com/bggardner/simplisafe-rf/blob/266d5e68d3e446fbbd7083f344c3688703a7d6cf/simplisafe/pigpio.py#L221

Should this be next_bit ^= 1 instead of next_bit ^= next_bit?. The equivalent section in the 'script' version of this method implies it should be.

**Comment by bggardner** [comment]
Yes, I believe you're right.  However, I no longer have the V2 hardware so I can't test it.
